java/component/keyple-plugin/remotese/src/main/java/org/eclipse/keyple/plugin/remotese/pluginse/RemoteSePluginImpl.java

```diff
@@ -187,7 +193,7 @@ void onReaderEvent(ReaderEvent event) throws KeypleReaderNotFoundException {
     }
 
     @Override
-    public void setParameter(String key, String value) throws IllegalArgumentException {
+    public void setParameter(String key, String value) {
```

Comment:
setParameter(String key, String value) is already implemented in AbstractSeProxyComponent